java/src/main/java/com/google/oak/remote_attestation/InsecureAttestationVerifier.java

```diff
@@ -35,8 +62,54 @@ public class InsecureAttestationVerifier implements AttestationVerifier {
    * @return success value wrapped in a {@code Result}
    */
   @Override
-  public final Result<Boolean, Exception> verify(
-      final AttestationEvidence evidence, final AttestationEndorsement endorsement) {
-    return Result.success(true);
+  public final Result<AttestationResults, Exception> verify(long nowUtcMillis,
+      final Evidence evidence, final Endorsements endorsements,
+      final ReferenceValues referenceValues) {
+    logger.severe(evidence.getApplicationKeys().getEncryptionPublicKeyCertificate().toString());
```

Comment:
Maybe add more context to this log. And should it really be severe?

Reply:
Oops, debug artiffact, sorry!